### PR TITLE
Revamp of the index page

### PR DESCRIPTION
**Pull request by GormFrank**
Total revamp of the index page to improve UI/UX on both Mobile and Desktop and make it look a little bit like the [WET-BOEW style guide](https://wet-boew.github.io/wet-boew-styleguide/index-en.html).

### CODE REVIEW

File: index-en.md [hunk 1]
```diff
@@ -1,263 +1,280 @@
...
+		<div class="col-xs-12 col-md-auto pull-right">
+			<p><a href="https://github.com/wet-boew/GCWeb/archive/v10.0.4.zip" class="btn btn-primary">Download GCWeb theme <strong>v10.1.0</strong></a><br />
+				<small>(<time>{{ page.dateModified | %F }}</time> - <a href="https://github.com/wet-boew/gcweb/releases/tag/v10.1.0">Release notes</a>)</small></p>
+		</div>
 	</div>
-	<div class="col-xs-12 col-md-auto pull-right">
-		<p><a href="https://github.com/wet-boew/GCWeb/archive/v10.1.0.zip" class="btn btn-primary">Download GCWeb theme <strong>v10.1.0</strong></a><br />
-			<small>(<time>{{ page.dateModified | %F }}</time> - <a href="https://github.com/wet-boew/gcweb/releases/tag/v10.1.0">Release notes</a>)</small></p>
+</div>
+<div class="container-fluid wb-inview show-none bar-demo" data-inview="nav-menu">
```
Comment (duboisp) [inline index-en.md:24]: `bar-demo` ????

File: index-fr.md [hunk 2]
```diff
@@ -1,262 +1,280 @@
...
+		<div class="col-xs-12 col-md-auto pull-right">
+			<p><a href="https://github.com/wet-boew/GCWeb/archive/v10.1.0.zip" class="btn btn-primary">Télécharger le thème <strong>GCWeb v10.1.0</strong></a><br />
+				<small>(<time>{{ page.dateModified | %F }}</time> - <a href="https://github.com/wet-boew/gcweb/releases/tag/v10.1.0">Note de version</a>)</small></p>
+		</div>
 	</div>
-	<div class="col-xs-12 col-md-auto pull-right">
-		<p><a href="https://github.com/wet-boew/GCWeb/archive/v10.1.0.zip" class="btn btn-primary">Télécharger le thème <strong>GCWeb v10.1.0</strong></a><br />
-			<small>(<time>{{ page.dateModified | %F }}</time> - <a href="https://github.com/wet-boew/gcweb/releases/tag/v10.1.0">Note de version</a>)</small></p>
+</div>
+<div class="container-fluid wb-inview show-none bar-demo" data-inview="nav-menu">
```
Comment (duboisp) [inline index-fr.md:24]: `bar-demo` ?

File: index-fr.md [hunk 3]
```diff
@@ -1,262 +1,280 @@
...
+		<li class="mrgn-lft-md">
+			<a href="#sitesglobal" class="btn btn-link text-white"><span aria-hidden="true" class="fas fa-globe mrgn-rght-md"></span>Global et de sites</a>
+		</li>
+		<li class="mrgn-lft-md">
+			<a href="#other" class="btn btn-link text-white"><span aria-hidden="true" class="fas fa-info-circle mrgn-rght-md"></span>Autre</a>
+		</li>
+		<li class="mrgn-lft-md">
+			<a href="#developper-pour-gcweb" class="btn btn-link text-white"><span aria-hidden="true" class="fas fa-code mrgn-rght-md"></span>Développer pour GCWeb</a>
+		</li>
+	</ul>
```
Comment (duboisp) [inline index-fr.md:64]: Broken anchor

**Review by duboisp — CHANGES_REQUESTED** [review]

File: index-fr.md (hunk 2 above)
Comment (GormFrank) [reply index-fr.md:24]: Oh lol copy and paste artifacts :-P

**Review by duboisp — APPROVED** [review]
Thats is great.

Thanks @GormFrank

**Pull request merged**